src/shared/services/commandInterpreterHelper.js

```diff
@@ -178,6 +180,42 @@ const availableCommands = [{
       })
   }
 }, {
+  name: 'style',
+  match: (cmd) => /^style(\s|$)/.test(cmd),
+  exec: function (action, cmdchar, put, store) {
+    const match = action.cmd.match(/:style\s*(\S.*)$/)
+    let param = match && match[1] ? match[1] : ''
+
+    if (param === '') {
+      // Todo: show popup
```

Comment:
Instead of nothing, could you add to the user gets some feedback. Maybe something like: https://github.com/neo4j/neo4j-browser/blob/3.0/src/shared/services/commandInterpreterHelper.js#L184